config/environments/production.rb

```diff
@@ -14,17 +16,17 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Enable Rack::Cache to put a simple HTTP cache in front of your application
-  # Add `rack-cache` to your Gemfile before enabling this.
-  # For large-scale production use, consider using a caching reverse proxy like
-  # NGINX, varnish or squid.
-  # config.action_dispatch.rack_cache = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
+  # Do we want to switch to the Terser gem?
```

Comment:
Is this a question for reviewers, or just an idea for future work? Maybe create a ticket vs adding comments to a file that may be forgotten about and lose context?

Reply:
It's a question based on stuff that popped up when I googled an issue I was having with this. What we have also works but some people recommended switching to this gem as an alternative.